api/sentry/utils.py

```diff
@@ -7,6 +7,25 @@
 
 from utils.datetime_utils import DatetimeUtils
 
+project_config = {
+    "firefox-ios": {
+        "title": ":testops-apple: iOS",
+        "looker_dashboard_url": (
+            "https://mozilla.cloud.looker.com/dashboards/2380"
+        ),
+        "confluence_report_url": (
+            "https://mozilla-hub.atlassian.net/wiki/spaces/"
+            "MTE/pages/1631911951/iOS+Health+Monitor+Report"
+        )
+    },
+    "fenix": {
+        "title": ":testops-android: Android"
```

Comment:
We can add `looker_dashboard_url` and `confluence_report_url` here in the future.